app/controllers/api/v1/users_controller.rb

```diff
@@ -9,13 +9,14 @@ def show
     render json: @user
   end
 
+  # fix this to handle updating of the user data
   def update
-    @user = User.find_by!(id: params[:id])
+    @user = User.find_by(id: params[:id])

```

Comment:
I think we want to keep the exclamation mark so that this returns a 404 if the user isn't found.

I also think we may need to do some validation here to make sure `is_suspended` is only updated by admins, but we will have to do that later.